ui/apps/platform/src/reducers/centralCapabilities.ts

```diff
@@ -0,0 +1,99 @@
+import { Reducer, combineReducers } from 'redux';
+import isEqual from 'lodash/isEqual';
+
+import { PrefixedAction } from 'utils/fetchingReduxRoutines';
+import { fetchCentralCapabilities, CentralServicesCapabilities } from 'services/MetadataService';
+
+// Types
+
+export type CentralCapabilitiesAction = PrefixedAction<
+    'config/FETCH_CENTRAL_CAPABILITIES',
```

Comment:
Can you review the naming convention for the prefix? Is it related to the name of the API endpoint?

For example, `central-capabilities` for /v1/central-capabilities in analogy:
* `config` for /v1/config
* `metadata` for /v1/metadata